src/ornament.ts

```diff
@@ -245,7 +273,7 @@ export class Ornament extends Modifier {
     let lineSpacing = 1;
 
     // Beamed stems are longer than quarter note stems, adjust accordingly
-    if (!isPlacedOnNoteheadSide && note.beam) {
+    if (!isPlacedOnNoteheadSide && note.getBeam()) {
```

Comment:
Don't we have a `hasBeam` which is better?

Reply:
done